sklearn/utils/__init__.py

```diff
@@ -297,9 +301,11 @@ def resample(*arrays, **options):
     --------
     :func:`sklearn.utils.shuffle`
     """
+
```

Comment:
Could you add a little example in the docstring to show the usage of `stratify`.
Otherwise LGTM